### PR TITLE
update htaccess, testing content negotiation

### PR DESCRIPTION
**Pull request by sreyfe**

### CODE REVIEW

File: iberz/.htaccess [hunk 1]
```diff
@@ -1,6 +1,28 @@
-AddType text/turtle .ttl
 
+# Do not remove this line, otherwise mod_rewrite rules will stop working
+RewriteBase /
```
Comment (davidlehn) [inline iberz/.htaccess:3]: Why would the rules stop working?  Rewrite base is used to reference local files and only remote redirects are used here.

File: iberz/.htaccess [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+
+# Rewrite engine setup<br/>
```
Comment (davidlehn) [inline iberz/.htaccess:5]:
```suggestion
# Rewrite engine setup
```

File: iberz/.htaccess [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
 
-RewriteRule ^(.*)$ https://iberz.org/$1 [R=302,L]
+
+AddType application/rdf+xml .owl
+
+AddType application/rdf+xml .rdf .owl
+
+AddType text/turtle .ttl
+
+# Rewrite rule to serve HTML content<br/>
```
Comment (davidlehn) [inline iberz/.htaccess:15]:
```suggestion
# Rewrite rule to serve HTML content
```

File: iberz/.htaccess (hunk 1 above)
Comment (sreyfe) [reply iberz/.htaccess:3]: just copied the code, I can delete the comment

File: iberz/.htaccess [hunk 4]
```diff
@@ -1,6 +1,28 @@
-AddType text/turtle .ttl
 
+# Do not remove this line, otherwise mod_rewrite rules will stop working
+RewriteBase /
+
```
Comment (davidlehn) [inline iberz/.htaccess:4]: Remove unneeded RewriteBase.
```suggestion
```

File: iberz/.htaccess (hunk 1 above)
Comment (davidlehn) [reply iberz/.htaccess:3]: Ok.  Added a suggestion. Feel free to test it out and make sure it does what you want.

Comment (sreyfe) [reply iberz/.htaccess]: looks good!

**Pull request merged**